quickwit/rest-api-tests/scenarii/sort_orders/0001-sort-elasticapi.yaml

```diff
@@ -10,10 +10,12 @@ json:
 expected:
   hits:
     total:
-      value: 5
+      value: 7
       relation: "eq"
     hits:
       - fields: { "count": 15, "id": 2 }
+      - fields: { "count": 10, "id": 0 }
```

Comment:
the bummer with those test is that they do not test against elasticsearch.

We need to add a set up for elastic like it is done in the es-compat-tests

Reply:
Yes they are just regular integration tests, but that's also fine I think